Makefile

```diff
@@ -119,9 +119,19 @@ setup-bare: clean e2e.namespace
 	. ./scripts/package_release.sh 1.0.0 test/e2e/resources test/e2e/e2e-bare-values.yaml
 	. ./scripts/install_bare.sh $(shell cat ./e2e.namespace) test/e2e/resources
 
-# e2e test exculding the rh-operators directory which tests rh-operators and their metric cardinality.
+# TODO(tflannag): Remove this target entirely and move downstream
 e2e:
-	go test -v $(MOD_FLAGS)  -failfast -timeout 150m ./test/e2e/... -namespace=openshift-operators -kubeconfig=${KUBECONFIG} -olmNamespace=openshift-operator-lifecycle-manager -dummyImage=bitnami/nginx:latest -ginkgo.flakeAttempts=3
+	$(GINKGO) \
+        ./test/e2e \
+        -flakeAttempts 3 \
```

Comment:
Let's take `flakeAttempts` as a variable and default to 1, that way we can change it with a downstream commit if we want.

Reply:
Done:
```
make e2e FLAKE_ATTEMPTS=9
GO111MODULE=on GOFLAGS="-mod=vendor" go run github.com/onsi/ginkgo/ginkgo \
        ./test/e2e \
        -flakeAttempts 9 \
        -randomizeAllSpecs \
        -v \
        -timeout 120m \
         -- \
        -namespace=openshift-operators \
        -kubeconfig=/home/tflannag/Downloads/cluster-bot-2021-09-21-132129.kubeconfig.txt \
        -olmNamespace=openshift-operator-lifecycle-manager \
        -dummyImage=bitnami/nginx:latest
```